src/sql/magic_plot.py

```diff
@@ -0,0 +1,79 @@
+from IPython.core.magic import (
+    Magics,
+    line_magic,
+    magics_class,
+)
+from IPython.core.magic_arguments import argument, magic_arguments
+
+
+try:
+    from traitlets.config.configurable import Configurable
+except ImportError:
+    from IPython.config.configurable import Configurable
+
+
+from sql import plot
+from sql.command import SQLPlotCommand
+
+
+@magics_class
+class SqlPlotMagic(Magics, Configurable):
+    """%sqlplot magic"""
+
+    @line_magic("sqlplot")
+    @magic_arguments()
+    @argument("line", default="", nargs="*", type=str, help="Plot name")
+    @argument("-t", "--table", type=str, help="Table to use")
+    @argument("-c", "--column", type=str, nargs="+", help="Column(s) to use")
```

Comment:
What about plotting specific rows? (kind of like a `pandas.df.head()/sample()`)

Reply:
currently, only boxplot and histogram are supported, those need entire columns for plotting